Yury_Novik/02.Git.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,143 @@
+# 1: Initializing a Local Repository
+
```

Comment:
please use code section for command history 
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks